Sources/SwiftUITooltip/TooltipConfigurations/ArrowOnlyTooltipConfig.swift

```diff
@@ -40,6 +40,8 @@ public struct ArrowOnlyTooltipConfig: TooltipConfig {
     public var animationOffset: CGFloat = 10
     public var animationTime: Double = 1
 
+    public var transition: AnyTransition = .opacity
```

Comment:
You would have to check this, but I think you can use `@Environment(\.transition) var transition: AnyTransition` in the TooltipModifier instead. I'm not sure if this would work within a `ViewModifier` but it's worth a try I think :)

Reply:
I tried this but `\.transition` is not available as an environment value.

Reply:
I see, then let's keep it as it is here!